kornia/geometry/transform/imgwarp.py

```diff
@@ -458,6 +458,8 @@ def remap(tensor: torch.Tensor, map_x: torch.Tensor, map_y: torch.Tensor,
         padding_mode (str): padding mode for outside grid values
           'zeros' | 'border' | 'reflection'. Default: 'zeros'.
         align_corners (bool, optional): mode for grid_generation. Default: None.
+        normalized_coordinates (bool): whether to normalize
+          coordinates in the range [-1, 1]
```

Comment:
`whether the input coordinates are normalised in the range of [-1, 1]`

Reply:
@edgarriba Shouldn't this be `whether the input coordinates **should be** normalised in the range of [-1, 1]`.
This obviously stands true if the argument there explicitly asks for normalization like I mentioned in the above comment instead of assuming it to be already normalized in which case it should be `if not normalized_coordinates`

Reply:
Summarizing my thoughts here. @edgarriba Let me know if I'm wrong or misdirected.

I assume that argument to be an explicit call to normalize the input because it isn't normalized already. Is this right? In this case I feel it should be `normalize_coordinates`.

Or should it be an indication that the argument means the input is already normalized and need not be normalized again. In this case it should be `normalized_coordinates`.

Reply:
@edgarriba @ducha-aiki Can you confirm one of these?